From: Matthias Clasen <mclasen@redhat.com>
Date: Mon, 27 Aug 2018 20:41:30 -0400
Subject: Flatpak portal: Documentation improvement

Explain what exposes are allowed.

Closes: https://github.com/flatpak/flatpak/issues/1955

Closes: #2029
Approved by: alexlarsson

(cherry picked from commit 1831ac19747109d299f6a3f86d285682c1bc42eb)
---
 data/org.freedesktop.portal.Flatpak.xml | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/data/org.freedesktop.portal.Flatpak.xml b/data/org.freedesktop.portal.Flatpak.xml
index b7bbed2..bc5b0bf 100644
--- a/data/org.freedesktop.portal.Flatpak.xml
+++ b/data/org.freedesktop.portal.Flatpak.xml
@@ -84,14 +84,16 @@
              <term>sandbox-expose as</term>
              <listitem><para>
                A list of filenames for files inside the sandbox that will be exposed
-               to the new sandbox, for reading and writing.
+               to the new sandbox, for reading and writing. Note that absolute paths
+               or subdirectories are not allowed.
              </para></listitem>
            </varlistentry>
            <varlistentry>
              <term>sandbox-expose-ro as</term>
              <listitem><para>
                A list of filenames for files inside the sandbox that will be exposed
-               to the new sandbox, readonly.
+               to the new sandbox, readonly. Note that absolute paths or subdirectories
+               are not allowed.
              </para></listitem>
            </varlistentry>
          </variablelist>
